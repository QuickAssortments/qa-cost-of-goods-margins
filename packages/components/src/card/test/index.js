/**
 * External dependencies
 *
 * @format
 */
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

/**
 * Internal dependencies
 */
import Card from '../';

describe( 'Card', () => {
	test( 'it renders correctly', () => {
		const tree = renderer.create( <Card title="A Card Example" /> ).toJSON();
		expect( tree ).toMatchSnapshot();
	} );

	test( 'should have correct title', () => {
		const card = <Card title="A Card Example" />;
		expect( card.props.title ).toBe( 'A Card Example' );
	} );

	test( 'should have correct class', () => {
		const card = shallow( <Card title="A Card Example" /> );
		expect( card.hasClass( 'woocommerce-card' ) ).toBe( true );
	} );
} );
