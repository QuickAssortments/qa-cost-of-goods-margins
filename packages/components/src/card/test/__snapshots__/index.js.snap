// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card it renders correctly 1`] = `
<div
  className="woocommerce-card"
>
  <div
    className="woocommerce-card__header"
  >
    <div
      className="woocommerce-card__title-wrapper"
    >
      <h2
        className="woocommerce-card__title woocommerce-card__header-item"
      >
        A Card Example
      </h2>
    </div>
  </div>
  <div
    className="woocommerce-card__body"
  />
</div>
`;
